### ISSUE TITLE
On the Optimization of Deep Networks: Implicit Acceleration by Overparameterization

### ISSUE THREAD

**Issue by Swall0w** [Swall0w/papers#585]
Sanjeev Arora, Nadav Cohen, Elad Hazan

Conventional wisdom in deep learning states that increasing depth improves expressiveness but complicates optimization. This paper suggests that, sometimes, increasing depth can speed up optimization. The effect of depth on optimization is decoupled from expressiveness by focusing on settings where additional layers amount to overparameterization – linear neural networks, a well-studied model. Theoretical analysis, as well as experiments, show that here depth acts as a preconditioner which may accelerate convergence. Even on simple convex problems such as linear regression with ℓp loss, p>2, gradient descent can benefit from transitioning to a non-convex overparameterized objective, more than it would from some common acceleration schemes. We also prove that it is mathematically impossible to obtain the acceleration effect of overparametrization via gradients of any regularizer.

http://proceedings.mlr.press/v80/arora18a/arora18a.pdf
